pkg/grpc/logging/logging.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // UnaryServerInterceptor returns a grpc.UnaryServerInterceptor that logs incoming requests
 // and associated tags to "logger".
-func UnaryServerInterceptor(logger *logging.Logger) grpc.UnaryServerInterceptor {
+func UnaryServerInterceptor(logger *logging.LoggerImpl) grpc.UnaryServerInterceptor {
```

Comment:
Similar to that, we can avoid using `logging.LoggerImpl` here by extending the interface to have a `SugardedLogger()` method which will return the sugared logger. This way we avoid exposing the interface's implementation and only have a single "thing" people may reference.